warehouse/static/js/warehouse/index.js

```diff
@@ -62,6 +62,21 @@ docReady(() => {
   }
 });
 
+// Hide webauthn option for unsupported browsers, or show errors
```

Comment:
Is fully hiding it the right move? I think graying it out would be more reasonable for discoverability of the feature.